backend/oasst_backend/api/v1/frontend_users.py

```diff
@@ -6,13 +6,30 @@
 from oasst_backend.api.v1 import utils
 from oasst_backend.models import ApiClient
 from oasst_backend.prompt_repository import PromptRepository
+from oasst_backend.user_repository import UserRepository
 from oasst_shared.schemas import protocol
 from sqlmodel import Session
 from starlette.status import HTTP_204_NO_CONTENT
 
 router = APIRouter()
 
 
+@router.get("/{auth_method}/{username}", response_model=protocol.User)
+def query_frontend_user(
+    auth_method: str,
+    username: str,
+    api_client_id: UUID = None,
```

Comment:
since it has `None` default it probably should be `Optional[UUID]` .. if it is optional there needs to be a fallback to the `api_client.id` somewhere (since the api-key can also be specified as header field and in this case the api_client_id param would be None but the api_client would be a correct ApiClient instance...)